app/views/portal/show.rb

```diff
@@ -36,13 +36,6 @@ def head_meta
       end
     end
 
-    def page_title
-      mustache[:page_title] ||= begin
-        title = [display_title, creator_title]
-        CGI.unescapeHTML(title.compact.join(' | ')) + ' - Europeana'
```

Comment:
This logic needs to still be kept around, otherwise all search result pages just have ' - Europeana Collections' as their title.
Following the convention the pages should be titled: 'Record Title  - Europeana Collections', so you may want to use  page_content_heading instead of page_title here.